blockstack/blockstackd.py

```diff
@@ -3795,6 +3795,19 @@ class Program {
             hash_out.write(json_hash)
             hash_out.flush()
 
+        placeholder_dict = {
```

Comment:
Just to be clear -- there's going to be an audit step somewhere that verifies that all of these (new) addresses have the appropriate tokens granted, right?  i.e. is there a spreadsheet somewhere outside the codebase that tracks who should have what address balance, for any special cases (cc @jessesoslow)?

Reply:
Yes, these addresses come from a spreadsheet created by Jesse S, not sure if the sheet is public at the moment. 

It sounds like we'll have an audit of the entire chainstate.txt export, including these ones.